ext/settings.html

```diff
@@ -1764,6 +1764,39 @@ <h1>Yomichan Settings</h1>
                 <label class="toggle"><input type="checkbox" data-setting="anki.suspendNewCards"><span class="toggle-body"><span class="toggle-track"></span><span class="toggle-knob"></span></span></label>
             </div>
         </div></div>
+        <div class="settings-item advanced-only">
+            <div class="settings-item-inner">
+                <div class="settings-item-left">
+                    <div class="settings-item-label">Note viewer window</div>
+                    <div class="settings-item-description">
+                        Clicking the <em>View added note</em> button shows this window.
+                        <a tabindex="0" class="more-toggle more-only" data-parent-distance="4">More&hellip;</a>
+                    </div>
+                </div>
+                <div class="settings-item-right">
+                    <select data-setting="anki.noteGuiMode">
+                        <option value="browse">Card browser</option>
+                        <option value="edit">Note editor</option>
+                    </select>
+                </div>
+            </div>
+            <div class="settings-item-children more" hidden>
+                <p>
+                    AnkiConnect releases after around 2022-05-20 support a new note editor window
```

Comment:
* [ ] TODO - update this date once this feature is actually deployed.

Reply:
Updated date based on merge date of https://github.com/FooSoft/anki-connect/pull/323. Note that this is not necessarily the release date of an AnkiConnect update.